spec/fixtures/daily.bai2

```diff
@@ -1,8 +1,8 @@
-01,121140399,9999999999,050608,1015,1,80,1,2/
-02,9999999999,121140399,1,050607,1015,,2/
-03,1234567890,,,,,/
-16,174,25001,Z,,50848,/
-88,SAMPLE CONTINUATION TEXT,/
-49,25001,4/
-98,25001,1,6/
+01,121140399,9999999999,050608,1015,1,80,1,2/
+02,9999999999,121140399,1,050607,1015,,2/
+03,1234567890,,,,,/
+16,174,25001,Z,,50848,/
+88,SAMPLE CONTINUATION TEXT,/
+49,25001,4/
+98,25001,1,6/
 99,25001,1,8/
```

Comment:
Should we add newline at end of file?
```suggestion
99,25001,1,8/

```

Reply:
Not sure, I didn't intentionally change anything. Is there meant to be one there?